translations/es.lang

```diff
@@ -78,25 +78,25 @@ lang_title=Idioma
 lang_desc=Idiomas disponibles:
 lang_thislang=Español
 lang_success=¡Idioma cambiado correctamente!
-hours=hours
-minutes=minutes
-seconds=seconds
-rank_rank=Rank
-rank_level=Level
-needs_manage_roles=You need to give the bot the Manage Roles permission to use this feature!
-rank_roles_title=Rank Roles
-rank_roles_setup=Use the ``rankrole`` command to setup Rank Roles.
-rank_role_title=Rank Role
-rank_role_usage=rankrole <config>. If you want to clear your Rank Roles, you can set config to ``none``. Config is a set of rules for your rank roles. For example, ``playtime_20:12345678912345678, sodas_10:12345678912345679`` will give the ``20`` people with the highest ``playtime`` the role with the ID ``12345678912345678``, then the ``10`` people with the most ``sodas`` the role ``12345678912345679``.
-rank_role_invalid_rule=This contains an invalid rule. Rules should follow the following format: ``metric_number:roleID``.
-rank_role_invalid_metric=Your config contains an invalid ``metric``. The following are valid metrics: ``kills``, ``deaths``, ``sodas``, ``medkits``, ``adrenaline``, ``rounds``, ``playtime``, ``escapes``, ``leaderboard``, ``fastestescape``.
-rank_role_invalid_number=Your config contains an invalid ``number``. Your ``number`` should be greater than zero but less than 100.
-rank_role_invalid_role=Your config contains an invalid role ID!
-rank_role_success=Successfully updated rank roles!
-warnings_title=Warnings
-warnings_footer=Warnings for 
-warnings_message_top=ID | Type | Message | Ban Length
-warnings_type_warning=Warning
+hours=horas
+minutes=minutos
+seconds=segundos
+rank_rank=Rango
+rank_level=Nivel
+needs_manage_roles=Necesitas dar al bot el permiso Gestionar Roles para usar esta característica!
+rank_roles_title=Roles de rango
+rank_roles_setup=Usa el comando ``rankrole`` para configurar los roles de rango.
+rank_role_title=Rol de rango
+rank_role_usage=rankrole <config>. Si quieres borrar los roles de rango, puedes poner la configuración en ``none``. La configuración es una serie de reglas para tus roles de rango. Por ejemplo, ``playtime_20:12345678912345678, sodas_10:12345678912345679`` daria a ``20`` personas con el mayor ``tiempo en el servidor`` el rol con la ID ``12345678912345678``, y a ``10`` personas con el mayor número de ``sodas`` el rol con ID ``12345678912345679``.
```

Comment:
Everything inside of backticks should be left untranslated because they are inputs for the bot. I would suggest you add some parentheses after it and put in the translation there.